corehq/apps/reports/standard/sms.py

```diff
@@ -707,6 +708,12 @@ def get_queryset(self):
                 Q(messagingsubevent__content_type__in=content_type_filter)),
         )
 
+        event_status = EventStatusFilter.get_value(self.request, self.domain)
+        if event_status is not None:
+            data = data.filter(
```

Comment:
Hmm, that kinda sucks. Is it not possible to update the main event level status during processing rather than have to do this complex filtering?